nomos-sdp/src/lib.rs

```diff
@@ -46,7 +52,7 @@ pub enum ServiceType {
 
 pub type Nonce = [u8; 16];
 
-#[derive(Clone, Copy, Eq, PartialEq, Hash, Debug)]
+#[derive(Clone, Copy, Eq, PartialEq, Hash, Debug, Serialize, Deserialize)]
```

Comment:
We need to take care of how this is done. Probably it should be transparent here, otherwise is gonna be wrapped.